configs/CSC-Lumi-HPECray-zen3-MI250X-Slingshot/variables.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+variables:
+  gtl_flag: ''  # to be overwritten by tests that need GTL
+  rocm_arch: 'gfx90a'
+  batch_time: '02:00'
+  mpi_command: 'srun -N {n_nodes} -n {n_ranks}'
+  batch_submit: 'sbatch {execute_experiment}'
+  batch_nodes: '#SBATCH -N {n_nodes}'
+  batch_ranks: '#SBATCH -n {n_ranks}'
+  batch_timeout: '#SBATCH -t {batch_time}:00'
+  cpu_partition: '#SBATCH -p small'
+  gpu_partition: '#SBATCH -p small-g' 
+  gpu_config: '#SBATCH --gpus-per-task=1'
```

Comment:
I'm not aware of any of our experiment templates making use of `gpu_config`. I'm not sure this is the way we'll want to allocate resources (I think whether we allocate 1 GPU per task is more relevant to a given experiment than to the whole system config).

Reply:
I agree, it can be removed.